docs/sdk-and-tools/sdk-js/sdk-js-cookbook-v13.md

```diff
@@ -0,0 +1,353 @@
+(whether deriving from the default ones or writing a new one, from scratch) that directly interacts with the MultiversX API (or Gateway).
+
+On this topic, please see [extending sdk-js](https://docs.multiversx.com/sdk-and-tools/sdk-js/extending-sdk-js).
+
+## Fetching network parameters
+
+```
+const networkConfig = await apiNetworkProvider.getNetworkConfig();
+console.log(networkConfig.MinGasPrice);
+console.log(networkConfig.ChainID);
+```
+
+## Working with accounts
+
+### Synchronizing an account object
+
+The following snippet fetches (from the Network) the **nonce** and the **balance** of an account, and updates the local representation of the account.
+
+```
+import { Account } from "@multiversx/sdk-core";
+
+const alice = new Account(addressOfAlice);
+const aliceOnNetwork = await apiNetworkProvider.getAccount(addressOfAlice);
+alice.update(aliceOnNetwork);
+
+console.log("Nonce:", alice.nonce);
+console.log("Balance:", alice.balance.toString());
+```
+
+### Managing the sender nonce locally
+
+When sending a bunch of transactions, you usually have to first fetch the account nonce from the network (see above), then manage it locally (e.g. increment upon signing & broadcasting a transaction):
+
+```
+alice.incrementNonce();
+console.log("Nonce:", alice.nonce);
+```
+
+:::note
+Since `sdk-core v13`, the `Transaction` class exhibits its state as public read-write properties. For example, you can access and set the `nonce` property, instead of using `getNonce` and `setNonce`.
+:::
+
+If you are using `sdk-core v13` or later, use `tx.nonce = ` to apply the nonce to a transaction. 
+For `sdk-core v12` or earlier, use the legacy `tx.setNonce()` to apply the nonce to a transaction.
+
+```
+
+notYetSignedTx.nonce = alice.getNonceThenIncrement();
+```
+
+For further reference, please see [nonce management](https://docs.multiversx.com/integrators/creating-transactions/#nonce-management).
+## Broadcasting transactions
+
+### Preparing a simple transaction
+
+:::note
+Since `sdk-core v13`, the `Transaction` class exhibits its state as public read-write properties. For example, you can access and set the `nonce` property, instead of using `getNonce` and `setNonce`.
+:::
+
+```
+import { Transaction, TransactionPayload } from "@multiversx/sdk-core";
+
+// Recommended approach: 
+
+const tx = new Transaction({
+    data: new TextEncoder().encode("food for cats"),
+    gasLimit: 70000n,
+    sender: addressOfAlice.toBech32(),
+    receiver: addressOfBob.toBech32(),
+    value: 1000000000000000000n,
+    chainID: "D"
+});
+
+tx.nonce = 42n;
+
+// Legacy approach: 
+
+const txLegacy = new Transaction({
+    data: new TransactionPayload("helloWorld"),
+    gasLimit: 70000,
+    sender: addressOfAlice,
+    receiver: addressOfBob,
+    value: "1000000000000000000",
+    chainID: "D"
+});
+
+txLegacy.setNonce(43);
+```
+
+### Broadcast using a network provider
```

Comment:
Perhaps also sign the transaction using `js-wallet`, so the user has the complete flow. Just an idea.

Reply:
Fixed, added.